website_sale_suggest_create_account/views/website_sale.xml

```diff
@@ -2,31 +2,32 @@
 <odoo>
     
     <template id="cart" inherit_id="website_sale.cart">
+        <xpath expr="//div[@id='wrap']" position="before">
+            <t t-set="user_authenticated" t-value="user_id != website.user_id"/>
+            <t t-set="signup_allowed" t-value="env['ir.config_parameter'].get_param('auth_signup.allow_uninvited') == 'True' "/>
+            <!-- note: optional_products has to be removed since odoo 12.0 at least. See https://github.com/odoo/odoo/pull/21461 -->
```

Comment:
I think the standard is to use "TODO" instead of "note". IDEs tend to be able to highlight those comments in a special way.